packages/fluentui/react-northstar/src/components/Tree/Tree.tsx

```diff
@@ -48,7 +48,7 @@ export interface TreeProps extends UIComponentProps, ChildrenComponentProps {
   exclusive?: boolean;
 
   /** Shorthand array of props for Tree. */
-  items?: ShorthandCollection<TreeItemProps>;
+  items?: TreeItemProps[];
```

Comment:
We need to support children as fn, so these typings are not quite correct. Take a look on the `ShorthandValue`, it supports alto this: `(Props<P> & { children?: P['children'] | ShorthandRenderFunction<P> })`. Maybe we should add a special typings for cases where we don't want to support react node, only objects and children as fn..

Reply:
Created `ObjectShorthandValue` and `ObjectShorthandCollection`